src/test/java/testresults/CVSB_495/ReviewTestSummary_2704.java

```diff
@@ -86,40 +88,44 @@ public void editingTestTypeResultAndGoingBackToTestReviewScreen() {
         //remove test type from change details screen
         testSteps.reviewAction();
         testReviewSteps.changeDetails();
-        testReviewSteps.pressTestTypeRemoveButton();
+        testTypeDetailsSteps.pressTestTypeRemoveButton();
         testSteps.pressRemoveInPopUp();
 
         //add a new test type
         testReviewSteps.goToTestPage();
         testSteps.addTestType();
         testTypeCategorySteps.selectFromTestTypeList("Annual test");
         testSteps.selectTestType("Annual test", TestPage.TestTypeStatuses.IN_PROGRESS);
-        testTypeDetailsSteps.setCarriedOutDuringTest(true);
+        testTypeDetailsSteps.setCarriedOutDuringTest(false);
         testTypeDetailsSteps.selectMostRecentInstallationCheck();
         testTypeDetailsSteps.setMostRecentInstallationCheckDateOneUnit();
         testTypeDetailsSteps.selectNumberOfSeatbeltsFitted();
         seatbeltInstallationCheckSteps.inputNumberOfSeatbelts("8");
         testTypeDetailsSteps.pressSave();
         testSteps.reviewAction();
         testReviewSteps.changeDetails();
-        //TODO uncomment after manual execution is done
+
         //edit the test type
-//        testTypeDetailsSteps.setCarriedOutDuringTest(true);
-//        testTypeDetailsSteps.selectMostRecentInstallationCheck();
-//        testTypeDetailsSteps.setMostRecentInstallationCheckDateOneUnit();
-//        testTypeDetailsSteps.inputNumberOfSeatbelt("16");
-//        testTypeDetailsSteps.pressSave();
+        testTypeDetailsSteps.setCarriedOutDuringTest(true);
+        testTypeDetailsSteps.selectNumberOfSeatbeltsFitted();
+        seatbeltInstallationCheckSteps.inputNumberOfSeatbelts("16");
 
         //Add defect from change details screen
-//        testTypeDetailsSteps.clickAddDefectButton();
-//        defectCategorySteps.selectDefectFromList("3. Seat Belts & Supplementary Restraint Systems");
-//        defectItemSteps.selectDefectFromList("1. Obligatory Seat Belt");
-//        defectDescriptionSteps.selectDefect("3.1");
-//        defectDetailsSteps.selectOptionsWithPRSCheckAndTapAddDefect("Upper", "Nearside", "2", "2");
+        testTypeDetailsSteps.clickAddDefect();
+        defectCategorySteps.selectDefectFromList("3. Seat Belts & Supplementary Restraint Systems");
+        defectItemSteps.selectDefectFromList("1. Obligatory Seat Belt");
+        defectDescriptionSteps.selectDefect("3.1");
+        defectDetailsSteps.setPRS();
+        HashMap<String, String> map = new HashMap<>();
```

Comment:
Please move this into the step class method.

Reply:
done